### PR TITLE
add head, body, content

### PR DESCRIPTION
**Pull request by borislavkb**
Added really basic structure (head, body) and some content. Not finished, still need to add Links to linkedIn/Xing and Bio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,5 +6,54 @@
...
-  <body></body>
+  <body>
+    <header>
+      <h1>Borislav Borislavov</h1>
+      <section class="topnav">
+        <a href="#contact">Contact</a>
+        <a href="#bio">Biography</a>
+      </section>
+    </header>
+    <br />
```
Comment (adrianammt) [inline index.html]: The br tag breaks text on a paragraph. Maybe you wanted to do something else?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

File: index.html [hunk 2]
```diff
@@ -6,5 +6,54 @@
...
+    <main>
+      <img
+        src="https://profile-images.xing.com/images/18f0cdb39a7fa0b8966ef3084f29ad92-1/borislav-borislavov.256x256.jpg"
+        alt="Profile picture from xing.com"
+      />
+      <br>
```
Comment (adrianammt) [inline index.html]: Same as line 17.

File: index.html [hunk 3]
```diff
@@ -6,5 +6,54 @@
...
+          <li>Beekeeping</li>
+      </ul>
+      <br>
+      <h2>Skills</h2>
+      <ul>
+          <li>a</li>
+          <li>b</li>
+          <li>c</li>
+      </ul>
+      <br>
```
Comment (adrianammt) [inline index.html]: Same as line 17.

File: index.html [hunk 4]
```diff
@@ -6,5 +6,54 @@
...
+      <section>
+          <!--Add a border to this section-->
+          <h2>Contact information</h2>
+          <p>Email: borislavkb@gmail.com</p>
+          <p>Mobile phone: 00 49 176 XXX XXXX</p>
+      </section>
+      <br>
```
Comment (adrianammt) [inline index.html]: Same as line 17.

File: index.html [hunk 5]
```diff
@@ -6,5 +6,54 @@
...
+      <section>
+          <a href="https://github.com/borislavkb" alt="">Link to Github</a>
+          <br>
```
Comment (adrianammt) [inline index.html]: Same as line 17.

File: index.html [hunk 6]
```diff
@@ -6,5 +6,54 @@
...
+      <img
+        src="https://profile-images.xing.com/images/18f0cdb39a7fa0b8966ef3084f29ad92-1/borislav-borislavov.256x256.jpg"
+        alt="Profile picture from xing.com"
+      />
+      <br>
+      <h2>Interests</h2>
+      <ul>
+          <li>Web development</li>
+          <li>Reading sci fi books</li>
+          <li>Beekeeping</li>
```
Comment (adrianammt) [inline index.html:30]: Nice!

File: index.html [hunk 7]
```diff
@@ -6,5 +6,54 @@
...
+      </ul>
+      <br>
```
Comment (adrianammt) [inline index.html:32]: Same as line 17.

File: index.html [hunk 8]
```diff
@@ -6,5 +6,54 @@
...
+      </section>
+      <br>
+      <section>
+          <a href="https://github.com/borislavkb" alt="">Link to Github</a>
+          <br>
+          <a href="url" alt="">Link to Codepen</a>
+      </section>
+
+
+      </p>
```
Comment (adrianammt) [inline index.html]: I can't find the opening tag for this paragraph.

File: index.html [hunk 9]
```diff
@@ -6,5 +6,54 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Personal website</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Borislav Borislavov</h1>
+      <section class="topnav">
+        <a href="#contact">Contact</a>
+        <a href="#bio">Biography</a>
```
Comment (mpagels) [inline index.html]: You can put the anchors in a list.
See more here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#examples

File: index.html [hunk 10]
```diff
@@ -6,5 +6,54 @@
...
+      </section>
+    </header>
+    <br />
+    <main>
```
Comment (mpagels) [inline index.html:20]: Very good to put the main content in the `main` element.

File: index.html [hunk 11]
```diff
@@ -6,5 +6,54 @@
...
+      <br>
+      <h2>Skills</h2>
+      <ul>
+          <li>a</li>
+          <li>b</li>
+          <li>c</li>
+      </ul>
+      <br>
+      <section>
+          <!--Add a border to this section-->
```
Comment (mpagels) [inline index.html]: You should remove this comment. I think this is just a mental note, but you don't want this in your production code.

File: index.html [hunk 12]
```diff
@@ -6,5 +6,54 @@
...
+          <h2>Contact information</h2>
+          <p>Email: borislavkb@gmail.com</p>
+          <p>Mobile phone: 00 49 176 XXX XXXX</p>
```
Comment (mpagels) [inline index.html]: Could be in a list as well.

File: index.html [hunk 13]
```diff
@@ -6,5 +6,54 @@
...
+      </section>
+      <br>
+      <section>
+          <a href="https://github.com/borislavkb" alt="">Link to Github</a>
```
Comment (mpagels) [inline index.html]: The `alt` attribute is only valid in an `` element. You must delete it here.

File: index.html [hunk 14]
```diff
@@ -6,5 +6,54 @@
...
+          <br>
+          <a href="url" alt="">Link to Codepen</a>
```
Comment (mpagels) [inline index.html]: And here. See line 46.

**Review by mpagels — CHANGES_REQUESTED** [review]
Hey Boris, very good. I found some little things. Please review and change some important suggestions.

File: index.html (hunk 1 above)
Comment (borislavkb) [reply index.html]: Thanks for the review and for the resource! :)))

File: index.html (hunk 9 above)
Comment (borislavkb) [reply index.html]: Put the anchors in a list ! Thanks ! 😃

File: index.html (hunk 2 above)
Comment (borislavkb) [reply index.html]: removed that. Basically I wanted to make the border of the break element visible but I assume this is not a good practice.

File: index.html (hunk 7 above)
Comment (borislavkb) [reply index.html:32]: '<br>' removed from line 30.

File: index.html (hunk 3 above)
Comment (borislavkb) [reply index.html]: removed the break element

File: index.html (hunk 11 above)
Comment (borislavkb) [reply index.html]: you are right, comment removed.

File: index.html (hunk 12 above)
Comment (borislavkb) [reply index.html]: Agree. Changed it to a list.

File: index.html (hunk 13 above)
Comment (borislavkb) [reply index.html]: ```alt``` attribute deleted

File: index.html (hunk 5 above)
Comment (borislavkb) [reply index.html]: resolved :)

File: index.html (hunk 14 above)
Comment (borislavkb) [reply index.html]: Edited

File: index.html (hunk 8 above)
Comment (borislavkb) [reply index.html]: Removed the ```</p>``` 😓

**Review by borislavkb — COMMENTED** [review]
Thank you for the comments. The proposed changes have been executed.

**Comment by borislavkb** [comment]
add some content, more content and links to be added.

**Review by borislavkb — COMMENTED** [review]
add requested changes

**Pull request merged**